Stream_HomeWork_1/src/Person.java

```diff
@@ -0,0 +1,28 @@
+
+public class Person {
+
+	String name;
```

Comment:
Always make class fields private and final:
`private final String name;`
because of immutability